@neutrinojs/fork error while running:  WebpackOptionsValidationError: Invalid configuration object

getting following error when using `@neutrinojs/fork` (8.0.9) for react & node

```
> nn_fork@1.0.0 start /home/ubuntu/projects/nn_fork
> neutrino start

✖ Building project failed
WebpackOptionsValidationError: Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration misses the property 'entry'.
   object { <key>: non-empty string | [non-empty string] } | non-empty string | [non-empty string] | function
   -> The entry point(s) of the compilation.
    at ChainAction.validator.chain.errors [as mapper] (/home/ubuntu/projects/nn_fork/node_modules/neutrino/src/webpack.js:49:20)
    at ChainAction$resolved [as resolved] (/home/ubuntu/projects/nn_fork/node_modules/fluture/index.js:798:27)
    at resolved (/home/ubuntu/projects/nn_fork/node_modules/fluture/index.js:213:19)
    at Resolved._fork (/home/ubuntu/projects/nn_fork/node_modules/fluture/index.js:636:3)
    at drain (/home/ubuntu/projects/nn_fork/node_modules/fluture/index.js:270:25)
    at settle (/home/ubuntu/projects/nn_fork/node_modules/fluture/index.js:199:15)
    at resolved (/home/ubuntu/projects/nn_fork/node_modules/fluture/index.js:213:5)
    at Immediate.EncaseP2$res (/home/ubuntu/projects/nn_fork/node_modules/fluture/index.js:1840:7)
    at runCallback (timers.js:791:20)
    at tryOnImmediate (timers.js:751:5)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! nn_fork@1.0.0 start: `neutrino start`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the nn_fork@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2017-12-10T07_47_12_696Z-debug.log

```
.neutrinorc.js
```
module.exports = {
  options: {},
  use: [
    [
      '@neutrinojs/fork', {
        configs: {
          react: '@neutrinojs/react',
          node: {
            options: { source: 'server', output: 'build/server' },
            use: ['@neutrinojs/node']
          }
        }
      }
    ]
  ]
};

```
on ubuntu 17.10, nodejs v8.9.3, npm 5.6.0, neutrinojs 8.0.9

--- a/packages/fork/index.js
+++ b/packages/fork/index.js
@@ -4,19 +4,22 @@ const { join } = require('path');
 const { createInterface } = require('readline');
 
 module.exports = (neutrino, options = {}) => {
-  const registeredCommand = neutrino.commands[neutrino.options.command];
-
   global.interactive = false;
 
-  neutrino.register(neutrino.options.command, (config, neutrino) =>
-    // The fork middleware was the only middleware used by Neutrino prior
-    // to running anything. We need to now prevent the default actions from
-    // doing anything else.
-    (Object.keys(config).length !== 0 ?
-      registeredCommand(config, neutrino) :
-      Promise.resolve()));
+  neutrino.on(`pre${neutrino.options.command}`, () => {
+    const registeredCommand = neutrino.commands[neutrino.options.command];
 
-  neutrino.on('prerun', () => {
+    neutrino.register(neutrino.options.command, (config, neutrino) => {
+      if (Object.keys(config).length === 0) {
+        // The fork middleware was the only middleware used by Neutrino prior
+        // to running anything. We need to now prevent the default actions from
+        // doing anything else.
+        return Promise.resolve();
+      }
+
+      return registeredCommand(config, neutrino);
+    });
+
     const children = Object
       .keys(options.configs || {})
       .map((namespace) => {

--- a/packages/fork/neutrino-child.js
+++ b/packages/fork/neutrino-child.js
@@ -40,7 +40,7 @@ process.on('message', ([rawMiddleware, args]) => {
     middleware.use.push(({ config }) => config.merge(args.options.config));
   }
 
-  const api = runnable(command, middleware, args);
+  const api = runnable(command, [middleware], args);
 
   api.on('*', (type, ...args) => {
     process.send([type, args]);

--- a/packages/neutrino/src/api.js
+++ b/packages/neutrino/src/api.js
@@ -267,9 +267,8 @@ class Api {
   // run :: String commandName -> Future
   run(commandName) {
     const emitForAll = this.emitForAll.bind(this);
-    const command = this.commands[commandName];
 
-    return Future((reject, resolve) => (command ?
+    return Future((reject, resolve) => (this.commands[commandName] ?
       resolve() :
       reject(new Error(`A command with the name "${commandName}" was not registered`))
     ))
@@ -279,7 +278,7 @@ class Api {
     .chain(() => Future.encaseP2(emitForAll, 'prerun', this.options.args))
     // Execute the command
     .chain(() => {
-      const result = command(this.config.toConfig(), this);
+      const result = this.commands[commandName](this.config.toConfig(), this);
 
       return Future.isFuture(result) ?
         result :

